examples/alpn/lib/http1_handler.ml

```diff
@@ -12,12 +12,12 @@ let redirect_handler : Unix.sockaddr -> Reqd.t Gluten.reqd -> unit =
   Reqd.respond_with_string reqd response ""
 
 let redirect_error_handler
-    :  Unix.sockaddr -> ?request:Request.t -> _ -> (Headers.t -> Body.Writer.t)
-    -> unit
+    :  Unix.sockaddr -> ?request:Request.t -> _
```

Comment:
Please revert this file. This is supposed to be http/af, not h2, as it is an HTTP/1.x handler.

Reply:
done